src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/indexed_fields_table/components/table/table.js

```diff
@@ -37,7 +39,7 @@ export class Table extends PureComponent {
     return value ? <EuiIcon type="dot" color="secondary" aria-label={label}/> : <span/>;
   }
 
-  renderFieldName(name, isTimeField) {
+  renderFieldName(name, isTimeField, label, content) {
```

Comment:
I think we can define `label` and `content` locally within this method, instead of requiring they be passed in as arguments. Same for `renderFieldType`, below. This will simplify the code a bit, and I think it will be truer to the intention of these methods as one-time helpers.